tests/scan/test_scan_report.py

```diff
@@ -51,3 +51,25 @@ def test_scan_report_exports_to_markdown():
         assert dest.exists()
         assert dest.is_file()
         assert dest.read_text() == markdown
+
+
+def test_scan_report_to_json():
```

Comment:
let's add a test where you pass the detector-names list to the scanreport, and check if the detector is in the json file